src/test/java/org/openstreetmap/atlas/checks/validation/linear/edges/MalformedRoundaboutCheckTest.java

```diff
@@ -0,0 +1,64 @@
+package org.openstreetmap.atlas.checks.validation.linear.edges;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * Tests for {@link MalformedRoundaboutCheck}
+ *
+ * @author savannahostrowski
+ */
+public class MalformedRoundaboutCheckTest
+{
+
+    private final MalformedRoundaboutCheck check = new MalformedRoundaboutCheck(
+            ConfigurationResolver.emptyConfiguration());
+    @Rule
+    public MalformedRoundaboutCheckTestRule setup = new MalformedRoundaboutCheckTestRule();
+
+    @Rule
+    public ConsumerBasedExpectedCheckVerifier verifier = new ConsumerBasedExpectedCheckVerifier();
+
+    @Test
+    public void testClockwiseRoundaboutLeftDrivingAtlas()
+    {
+        this.verifier.actual(this.setup.clockwiseRoundaboutLeftDrivingAtlas(), check);
+
+        this.verifier.verifyEmpty();
+    }
+
+    @Test
+    public void testClockwiseRoundaboutRightDrivingAtlas()
+    {
+        this.verifier.actual(this.setup.clockwiseRoundaboutRightDrivingAtlas(), check);
+
+        this.verifier.globallyVerify(flags -> Assert.assertEquals(1, flags.size()));
```

Comment:
I'd suggest checking more than just the count. Maybe the feature flagged.

Reply:
@mertemin Do you mean having each unit test verify the identifier of the edge that's been flagged? If that's the case, I'm not sure how that would improve the unit tests as it'll always be the first edge in the test Atlas, as that edge is always the starting edge for the roundabout?

Reply:
Cool, I am OK with leaving as-is.